docs/source/common/lightning_module.rst

```diff
@@ -983,6 +983,20 @@ The current logger being used (tensorboard or other supported logger)
         # the particular logger
         tensorboard_logger = self.logger.experiment
 
+loggers
+~~~~~~
+
+The list of loggers currently being used (tensorboard or other supported loggers)
+
+.. code-block:: python
+
+    def training_step(self, batch, batch_idx):
+        # the list of generic loggers (same no matter if tensorboard or other supported loggers)
+        self.loggers
+
+        # the particular logger
+        tensorboard_logger = self.loggers[0].experiment
```

Comment:
i don't think we should be highlighting self.loggers[0].experiment. it should be obvious that these are a list of loggers . especially now that experiment is not a required argument on the logger interface, this is just more confusing for new users.